Mathlib/RingTheory/PowerSeries/Restricted.lean

```diff
@@ -0,0 +1,147 @@
+/-
+Copyright (c) 2025 William Coram. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: William Coram
+-/
+
+import Mathlib.Algebra.Group.NatPowAssoc
+import Mathlib.Analysis.Normed.Group.Ultra
+import Mathlib.Analysis.RCLike.Basic
+import Mathlib.RingTheory.PowerSeries.Basic
+
+/-!
+# Restricted power series
+
+We say a powerseries over a normed ring `R` is restricted for a parameter `c` if
+`‖coeff R i f‖ * c^i → 0`.
+
+-/
+
+namespace PowerSeries
+
+open PowerSeries Filter
+open scoped Topology
+
+/-- A power series over `R` is restricted of paramerter `c` if we have the following limit. -/
```

Comment:
Please write the actual def in the docstring, rather than "the following limit".